Python/CarGame.py

```diff
@@ -0,0 +1,32 @@
+# A fun and interactive Python Car Game
+
```

Comment:
hmm, interesting game. Do the following things now:
- Remove the extra spaces here 
- Update the Python folder readme with your entry
- Add a screenshot of the game for one play, place both the files (code and image) in a folder and place that folder under python directory

Reply:
**Done**